apps/src/blockly/addons/cdoRendererZelos.js

```diff
@@ -0,0 +1,13 @@
+import GoogleBlockly from 'blockly/core';
+import CdoPathObject from './cdoPathObjectZelos';
+
+export default class CdoRenderer extends GoogleBlockly.zelos.Renderer {
```

Comment:
Rename this class?

Reply:
Good call.  Will do similar for `CdoPathObject` -> `CdoPathObjectZelos` too.